runelite-client/src/main/java/net/runelite/client/plugins/gpu/GpuPluginConfig.java

```diff
@@ -74,11 +75,22 @@ default AntiAliasingMode antiAliasingMode()
 	@Range(
```

Comment:
This range annotation should be on the fog depth not on the anisotropic filtering mode

Reply:
Ah okay, this looks like a minor thing. However, as I am a bit too busy to revisit this and you have a separate pull request going, I will probably only fix this if it is completely necessary to do so.